guides/lagom-sbr-conductr/lagom-scala-sbt/hello-lagom-impl/src/main/scala/com/example/hello/impl/HellolagomEntity.scala

```diff
@@ -0,0 +1,191 @@
+package com.example.hello.impl
+
+import java.time.LocalDateTime
+
+import akka.Done
+import com.lightbend.lagom.scaladsl.persistence.{AggregateEvent, AggregateEventTag, PersistentEntity}
+import com.lightbend.lagom.scaladsl.persistence.PersistentEntity.ReplyType
+import com.lightbend.lagom.scaladsl.playjson.{JsonSerializer, JsonSerializerRegistry}
+import play.api.libs.json.{Format, Json}
+
+import scala.collection.immutable.Seq
+
+/**
+  * This is an event sourced entity. It has a state, [[HellolagomState]], which
+  * stores what the greeting should be (eg, "Hello").
+  *
+  * Event sourced entities are interacted with by sending them commands. This
+  * entity supports two commands, a [[UseGreetingMessage]] command, which is
+  * used to change the greeting, and a [[Hello]] command, which is a read
+  * only command which returns a greeting to the name specified by the command.
+  *
+  * Commands get translated to events, and it's the events that get persisted by
+  * the entity. Each event will have an event handler registered for it, and an
+  * event handler simply applies an event to the current state. This will be done
+  * when the event is first created, and it will also be done when the entity is
+  * loaded from the database - each event will be replayed to recreate the state
+  * of the entity.
+  *
+  * This entity defines one event, the [[GreetingMessageChanged]] event,
+  * which is emitted when a [[UseGreetingMessage]] command is received.
+  */
+class HellolagomEntity extends PersistentEntity {
```

Comment:
`HellolagomEntity` => `HelloLagomEntity `?

Reply:
Thanks @ktoso - I agree. I'd prefer not to change it here though as we're just using the standard Lagom getting started template.

@TimMoore FYI.

Reply:
I think that's a function of how Giter8 applies your project name to the template.

Reply:
Yes, it assumes that you'll separate words with spaces and not hyphens: https://github.com/foundweekends/giter8/blob/29d329bb92977a7848e8b48bd2ba5d1cf551e210/library/src/main/scala/g8.scala#L202